webrender/src/border.rs

```diff
@@ -385,7 +385,7 @@ impl FrameBuilder {
                 self.add_solid_rectangle(
                     clip_and_scroll,
                     &info,
-                    &border.top.color,
+                    &FillOrClear::Fill(border.top.color),
```

Comment:
"XxxxOrYyy" pattern sounds a bit awkward to me. Perhaps, we could go with something like `Fill::Color` versus `Fill::Clear` ?